demo/components/tilt-card/default.tsx

```
@@ -1,28 +1,25 @@
-      </div>
-    </TiltCard>
-  )
-}
+    return (
+        <TiltCard className="bg-card w-64 rounded border p-4 sm:w-80" highlightClassName="bg-white/15 dark:bg-white/2">
+            <p className="text-muted-foreground font-mono text-xs">09, May 2025</p>
+            <p className="mt-1 text-lg/none font-medium">New York, USA</p>
+            <img
+                className="mt-3 h-40 w-full rounded object-cover"
+                src="https://raw.githubusercontent.com/ServiceStack/images/refs/heads/master/hero/black-white-city.jpg"
+                alt="Image"
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)